### PR TITLE
COL-429 Postgres migration for new activity types and objects

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/COL-429

### CODE REVIEW

File: scripts/20160812-col429/add_activity_enums.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+-- Add new activity enums
+
+ALTER TYPE enum_activities_type ADD VALUE 'view_asset';
+ALTER TYPE enum_activities_type ADD VALUE 'whiteboard_add_asset';
+ALTER TYPE enum_activities_type ADD VALUE 'whiteboard_chat';
+
+-- Add new activity type enums
+
+ALTER TYPE enum_activity_types_type ADD VALUE 'view_asset';
```
Comment (nicolaasmatthijs) [inline scripts/20160812-col429/add_activity_enums.sql]: @paulkerschen : Do you know what the difference is between `enum_activities_type` and `enum_activity_types_type` ?

**Comment by nicolaasmatthijs** [comment]
Left a question. Back to @paulkerschen

File: scripts/20160812-col429/add_activity_enums.sql (hunk 1 above)
Comment (paulkerschen) [inline scripts/20160812-col429/add_activity_enums.sql]: @nicolaasmatthijs Yes, they refer to different tables. Updated comments to clarify.

**Comment by paulkerschen** [comment]
Updated comments, back to @nicolaasmatthijs

**Comment by nicolaasmatthijs** [comment]
Thanks for the clarifications. Merged.

**Pull request merged**